fava/api/__init__.py

```diff
@@ -166,6 +166,9 @@ def _journal(self, postings, include_types=None, with_change_and_balance=False):
 
             entry = serialize_entry(posting)
 
```

Comment:
This doesn't belong in `_journal`...

It should go in `serialize_entry` or even better, just [add a filter](http://flask.pocoo.org/docs/0.10/templating/#registering-filters) and then do it in the template.